shoper.py

```diff
@@ -0,0 +1,135 @@
+import os
+import pandas as pd
+from operator import itemgetter
+from re import search
+
+
+def deactivate_products(database, dc_price, replacement_price):
+    """
+    Deactivates products set with exact price - e.g. 107.99
+    Products are kept because some of them have been withdrawn,
+    yet they might come back
+    
+    csv_file: .csv file to work with
```

Comment:
Docstringi tworzy się w odpowiednim formacie, tj.
```
"""
Possible comment explaining something.

:param database: (type) what is it
:param dc_price: (type) what is it
:return: (type) something something
"""
```
Please note it's also possible to change (type) location e.g.
```
:param database (type): explantion
```